### DISCUSSION TITLE
Make sure you have no overlapping segments / No segments to export

### DISCUSSION THREAD

**Discussion by HubGid** [mifi/lossless-cut#1078] [Q&A]
Nearly 16 hour long mp3.
I've used Edit>Segment>Create Num segments.
Set to 4

Four segments show correctly.
Of the side panel it always shows "Make sure you have no overlapping segments"
Is that an error or just a general reminder?

Just clicking export always gives: "No segments to export"
I'm so new at this that I don't even know how to select segments....
I configured a  "Select All" key binding but that doesn't seem to change anything.

Can someone please help this newbie getting started?

**Comment by mifi** [comment]
hi. could you share the .llc file right after you did "Create num segments"? also if you could paste the report here (Help->report)

**Comment by HubGid** [comment]
[MP3-split-test-proj.zip](https://github.com/mifi/lossless-cut/files/8283509/MP3-split-test-proj.zip)

No error

{
  "state": {
    "filePath": "E:\\---[ 0-days ]---\\!Bible\\!Books - Religious\\Michael S. Heiser\\The unseen realm\\MP3-split-test.mp3",
    "fileFormat": "mp3",
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "mp3",
        "codec_long_name": "MP3 (MPEG audio layer 3)",
        "codec_type": "audio",
        "codec_tag_string": "[0][0][0][0]",
        "codec_tag": "0x0000",
        "sample_fmt": "fltp",
        "sample_rate": "44100",
        "channels": 2,
        "channel_layout": "stereo",
        "bits_per_sample": 0,
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/14112000",
        "start_pts": 0,
        "start_time": "0.000000",
        "duration_ts": 799273081656,
        "duration": "56637.831750",
        "bit_rate": "32000",
        "disposition": {
          "default": 0,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0
        }
      }
    ],
    "copyStreamIdsByFile": {
      "E:\\---[ 0-days ]---\\!Bible\\!Books - Religious\\Michael S. Heiser\\The unseen realm\\MP3-split-test.mp3": {
        "0": true
      }
    },
    "cutSegments": [
      {
        "start": 0,
        "end": 14159.4579375
      },
      {
        "start": 14159.4579375,
        "end": 28318.915875
      },
      {
        "start": 28318.915875,
        "end": 42478.373812499995
      },
      {
        "start": 42478.373812499995
      }
    ],
    "mainFileFormatData": {
      "filename": "E:\\---[ 0-days ]---\\!Bible\\!Books - Religious\\Michael S. Heiser\\The unseen realm\\MP3-split-test.mp3",
      "nb_streams": 1,
      "nb_programs": 0,
      "format_name": "mp3",
      "format_long_name": "MP2/3 (MPEG audio layer 2/3)",
      "start_time": "0.000000",
      "duration": "56637.831750",
      "size": "226551359",
      "bit_rate": "32000",
      "probe_score": 51,
      "tags": {
        "encoded_by": "Lavf51.12.2"
      }
    },
    "rotation": 360,
    "shortestFlag": false
  },
  "platform": "win32",
  "version": "3.44.0"
}

**Comment by mifi** [comment]
Thanks. You need to click the YinYang symbol to enable normal cut mode. I will improve the error message to make that more clear

**Comment by HubGid** [comment]
Work like a charm!
And fast.
Thanks for the quick and very helpful support mifi.
